src/appleseed/renderer/kernel/lighting/bdpt/bdptlightingengine.cpp

```diff
@@ -83,37 +123,317 @@ namespace
         {
         }
 
+        Spectrum ComputeGeometryTerm(
```

Comment:
`compute_geometry_term()` according to our coding conventions:

    https://github.com/appleseedhq/appleseed/wiki/Coding-Philosophy-and-Guidelines#naming-conventions